### DISCUSSION TITLE
HAA-XXXXXX WiFi hotspot does not appear after flash fullhaaboot.bin

### DISCUSSION THREAD

**Discussion by epicsepi** [RavenSystem/esp-homekit-devices#1682] [General]
I did everything the way I did before, but the WiFi hotspot does not appear after successfull flash. 
I don’t know what the problem might be, but I can’t solve it for days. I flashed other devices, there was no problem with them. 

I noticed the unusual thing that when I solder cables to GND, VCC, RX, and TX, I don’t need to connect GPIO0 to GND. It can be erased or flashed without it. From this, I conclude that whenever I plug in the device, it always goes into flash mode automatically, which causes the hotspot to not appear.

Device: Moes universal IR controller with ESP chip
HAA release: 9.5.1 Griffon

![IMG_8320](https://user-images.githubusercontent.com/73914360/147940713-6ad0614b-4f9e-4164-aff6-c147165fb8a7.jpeg)

Please help me to solve this ISSUE!

Best regards,
Adam

**Comment by wahidulislamriyad** [comment]
Try flashing using any other flash tool.
I had same issue when I flashed few ESP8266 with ESP Flash Download tool. But when I flashed using another software called “esp-tool-gui” then it worked.
